<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StylePlannerClass</class>
 <widget class="QMainWindow" name="StylePlannerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>622</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StylePlanner</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background: url(C:/Users/Studia/Desktop/AplikacjaOkienkowaJIPP/bin/wood_background.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Zakladki">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>621</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabWidget::pane {
  border: 0px;
  top: 1px; 
} 

QTabBar::tab {
  border: 1px solid lightgray; 
  padding: 5px;
  background: url(C:/Users/Studia/Desktop/AplikacjaOkienkowaJIPP/bin/wood_tab.jpg);

} 

QTabBar::tab:selected { 
  margin-bottom: 1px;
  background: url(C:/Users/Studia/Desktop/AplikacjaOkienkowaJIPP/bin/wood_tab_open.jpg); 

}
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::East</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset theme="edit-select-all"/>
     </attribute>
     <attribute name="title">
      <string>‎ ‎ ‎ ‎ ‎ </string>
     </attribute>
     <widget class="QPushButton" name="PbutB">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>390</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-seek-forward"/>
      </property>
     </widget>
     <widget class="QPushButton" name="addBot">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="list-add"/>
      </property>
     </widget>
     <widget class="QPushButton" name="delBot">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="edit-delete"/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="LbutB">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>390</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-seek-backward"/>
      </property>
     </widget>
     <widget class="QPushButton" name="addTop">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>200</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="list-add"/>
      </property>
     </widget>
     <widget class="QLabel" name="BotPic">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="delTop">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="edit-delete"/>
      </property>
     </widget>
     <widget class="QLabel" name="TopPic">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Lbut">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-seek-backward"/>
      </property>
     </widget>
     <widget class="QPushButton" name="Pbut">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-seek-forward"/>
      </property>
     </widget>
     <widget class="QPushButton" name="flagFav">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="emblem-favorite"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset theme="emblem-favorite"/>
     </attribute>
     <attribute name="title">
      <string>‎ ‎ ‎ ‎ ‎ ‎</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>841</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea {
    background-color: rgb(180, 119, 66);
}

QScrollBar {
    background-color: rgb(221, 221, 221);
}
QWidget
{
 background-color: rgb(180, 119, 66);
}
QScrollBar:vertical { background-color: rgb(221, 221, 221); width: 15px; }
QScrollBar::handle:vertical { background-color: rgb(160, 160, 160); }
QScrollBar:horizontal { background-color: rgb(221, 221, 221); height: 15px; }
QScrollBar::handle:horizontal { background-color: rgb(160, 160, 160); }</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollCont">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>839</width>
         <height>579</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StylePlanner.qrc"/>
 </resources>
 <connections/>
</ui>
